src/Microsoft.DotNet.Build.Tasks/Targets/test-runtime/project.json

```diff
@@ -32,10 +32,12 @@
        "System.Threading": "4.0.10-beta-*",
        "System.Threading.Tasks": "4.0.10-beta-*",
        "System.Threading.ThreadPool": "4.0.10-beta-*",
+       "System.Threading.Thread": "4.0.0-beta-*",
        "System.Xml.ReaderWriter": "4.0.10-beta-*",
        "System.Xml.XDocument": "4.0.10-beta-*",
-       "xunit.console.netcore": "1.0.2-prerelease-00036",
-       "xunit.runner.dependencies.netcore": "1.0.1-prerelease"
+       "xunit": "2.1.0-beta3-*",
```

Comment:
I'm not convinced we want to use floating versions for these. 

@ericstj @nguerrera thoughts?